Code/Editor/ViewportManipulatorController.cpp

```diff
@@ -72,13 +74,23 @@ namespace SandboxEditor
                 const auto* position = event.m_inputChannel.GetCustomData<AzFramework::InputChannel::PositionData2D>();
                 AZ_Assert(position, "Expected PositionData2D but found nullptr");
 
+                if (m_captureStart)
+                {
+                    m_virtualPosition = position->m_normalizedPosition;
+                    m_captureStart = false;
+                }
+                else
+                {
+                    m_virtualPosition += position->m_normalizedPositionDelta;
+                }
+
                 AzFramework::WindowSize windowSize;
                 AzFramework::WindowRequestBus::EventResult(
                     windowSize, event.m_windowHandle, &AzFramework::WindowRequestBus::Events::GetClientAreaSize);
 
                 const auto screenPoint = AzFramework::ScreenPoint(
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetX() * windowSize.m_width),
-                    aznumeric_cast<int>(position->m_normalizedPosition.GetY() * windowSize.m_height));
+                    aznumeric_cast<int>(m_virtualPosition.GetX() * windowSize.m_width),
+                    aznumeric_cast<int>(m_virtualPosition.GetY() * windowSize.m_height));
```

Comment:
update: Just spotted this but can you update this to use `AzFramework::ScreenPointFromVector2`, it uses `AZStd::lround` to make sure `5.99999f` becomes `6.0f` instead of `5.0f`. So I think this would be...

```
const auto screenPoint =
    AzFramework::ScreenPointFromVector2(AZ::Vector2(
        m_virtualPosition.GetX() * windowSize.m_width,
        m_virtualPosition.GetY() * windowSize.m_height));
```